JitsiConference.js

```diff
@@ -3261,6 +3261,26 @@ JitsiConference.prototype.setReceiverVideoConstraint = function(
     this.rtc.setReceiverVideoConstraint(maxFrameHeight);
 };
 
+/**
+ * Sets the maximum video size the local participant should send to remote
+ * participants.
+ * @param {number} maxFrameHeight - The user preferred max frame height.
+ * @returns {Promise} promise that will be resolved when the operation is
+ * successful and rejected otherwise.
+ */
+JitsiConference.prototype.setSenderVideoConstraint = function(maxFrameHeight) {
+    const promises = [];
+
+    if (this.jvbJingleSession) {
+        promises.push(this.jvbJingleSession.setSenderVideoConstraint(maxFrameHeight));
+    }
+    if (this.p2pJingleSession) {
```

Comment:
So I don't think this is going to work out well. P2P connection comes and goes, so you will have the instance here only by luck.

I think you also need to store the `maxFrameHeight` value in the `JitsiConference` or in a dedicated submodule and then whenever JVB or P2P session is created apply the setting in case it didn't exist at the time this method was called.

Reply:
Thanks for pointing it out, I see how it can fail in some cases.